app/code/Magento/CatalogUrlRewrite/Model/Product/Attribute/Backend/UrlKey.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
Please, move the strict types declaration beyond the namespace declaration and add Magento copyright.